script.js

```diff
@@ -63,21 +63,39 @@ function renderTodos() {
 
   filteredTodos.forEach((todo, index) => {
     const listItem = document.createElement("li");
+    listItem.style.display = "flex";
```

Comment:
this is inline styling actually which complicates specificity and is a bad practice IMHO. next time you open your css file you won't find that there, but it still has an effect on styling.

Reply:
Move inline styling to css file.